jenkins-scripts/dsl/ignition.dsl

```diff
@@ -222,9 +222,9 @@ ignition_software.each { ign_sw ->
       abi_job_names[ign_sw] = "ignition_${ign_sw}-abichecker-any_to_any-ubuntu_auto-${arch}"
       def abi_job = job(abi_job_names[ign_sw])
       checkout_subdir = "ign-${ign_sw}"
-      OSRFLinuxABIGitHub.create(abi_job, checkout_subdir)
-      GenericAnyJobGitHub.create(abi_job,
-                        "ignitionrobotics/ign-${ign_sw}",
+      repo = "ignitionrobotics/ign-${ign_sw}"
+      OSRFLinuxABIGitHub.create(abi_job, repo)
```

Comment:
I'm not sure why the second argument is provided to `OSRFLinuxABIGitHub.create` here, since only `abi_job` is passed in [gazebo.dsl](https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/dsl/gazebo.dsl#L92) and [sdformat.dsl](https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/dsl/sdformat.dsl#L40)

Reply:
The method has an overloaded `create` that took two parameters and setup github checkout before calling the `create` used in gazebo and sdformat. I've removed that logic in 575590f to simplify the use and don't fall in these kind of errors again. The class should be renamed after #229 since it has nothing to do with GitHub.